scalableminds/webknossos
# Add Scalebar to 3d viewport

This PR adds the scalebar to the 3d viewport.

### URL of deployed dev instance (used for testing):
- https://scalebarfordviewport.webknossos.xyz

### Steps to test:
- open a tracing. Set the 3d Viewport to show the xy plane via the buttons in the to right corner.
- Then adjust the zoom in the settings so that the scalebar of the 3d viewport and the xy viewport have the same scale.
- The xy viewport and the 3d viewport should now display the (almost) exact same data. This should work for every zoom and every dataset.

### Issues:
- fixes #4223

------
